backend/prisma/schema.prisma

```diff
@@ -4,8 +4,8 @@ generator client {
 
 datasource db {
   provider = "postgresql"
-  url      = env("DATABASE_URL")
-  // url      = "postgresql://postgres:docker@localhost:5432/urefer?schema=public"
+  // url      = env("DATABASE_URL")
+  url      = "postgresql://postgres:docker@localhost:5432/urefer?schema=public"
```

Comment:
We probably want to keep it as env("DATABASE_URL")

Reply:
got it